.github/workflows/deploy-dev.yml

```diff
@@ -1,9 +1,6 @@
-name: Deploy to Dev
+name: Deploy to Dev (ECS)
 
-on:
-  workflow_dispatch:
-  push:
-    branches: [master]
```

Comment:
Ah, the

```
push:
  branches: [master]
```

shouldn't have been removed. That's the workflow trigger which causes api-dev to be automatically deployed to when we merge a PR, which is behavior we like in our dev environments.

Reply:
Completed the change 👍